### PR TITLE
python3Packages.brother: init at 0.1.14

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

Enable the [brother integration](https://www.home-assistant.io/integrations/brother/) in home-assistant.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by lsix — COMMENTED** [review]
nix-review gives me that:

```bash
[8 built (2 failed), 105 copied (197.3 MiB), 43.9 MiB DL]
error: build of '/nix/store/wark1gyjszmbycf8xw9k1sn9cm5s2cbz-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/94568
2 packages failed to build:
python27Packages.pytest-tornasync python37Packages.brother

5 packages built:
python37Packages.pytest-tornasync python37Packages.pytest-trio python38Packages.brother python38Packages.pytest-tornasync python38Packages.pytest-trio
```

I did not look into it, but it should be looked into or some packages should probably be disabled for some python versions (if this is the problem).

**Comment by mweinelt** [comment]
I set pytest-toransync to >=3.5.

The python3.7 version of brother is failing in asynctest, which has had its test disabled for 3.8, which is apparently also necessary for 3.7, see 25b91c2059b04268e70c5d73aac661f85bef48f6.

**Comment by mweinelt** [comment]
Disabled the asynctest tests on python 3.7 in #94655. Thinks are looking good now afaict.

File: pkgs/development/python-modules/pytest-tornasync/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+, pytest, tornado }:
+
+buildPythonPackage rec {
+  pname = "pytest-tornasync";
+  version = "0.6.0.post2";
```
Comment (jonringer) [inline pkgs/development/python-modules/pytest-tornasync/default.nix:6]:
```suggestion
  version = "0.6.0.post2";
  disabled = isPy27;
```
builder for '/nix/store/1qj7cyp9723msq2vl1lzq5wgwgm926ar-python2.7-pytest-tornasync-0.6.0.post2.drv' failed with exit code 1; last 10 log lines:
  unpacking sources
  unpacking source archive /nix/store/lzzb7w2hxgaji81yf3426308lca56awh-source
  source root is source
  setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tox.ini
  patching sources
  configuring
  no configure script, doing nothing
  building
  Executing setuptoolsBuildPhase
  pytest-tornasync requires Python 3.5 or newer
```

Comment (mweinelt) [reply pkgs/development/python-modules/pytest-tornasync/default.nix:6]: Hah, weird. I tought I already pushed that back when I remarked about it in https://github.com/NixOS/nixpkgs/pull/94568#issuecomment-668057659. :tired_face:

Comment (mweinelt) [reply pkgs/development/python-modules/pytest-tornasync/default.nix:6]: Done, thanks!

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 94568 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package failed to build:</summary><br>- python27Packages.pytest-tornasync</details><details>  <summary>6 packages built:</summary><br>- python37Packages.brother<br>- python37Packages.pytest-tornasync<br>- python37Packages.pytest-trio<br>- python38Packages.brother<br>- python38Packages.pytest-tornasync<br>- python38Packages.pytest-trio</details>

**Pull request merged**